box-ui/src/components/ts/DashboardButtonsComponent.tsx

```diff
@@ -1,82 +1,57 @@
+                <Popup
+                    trigger={
+                        <div>
+                            <Button variant='contained' className='Button' color='error' sx={{ borderRadius: 28 }}>
+                                <ExitToAppOutlinedIcon />
+                            </Button>
+                            Disconnect
+                        </div>
+                    }
+                    modal
+                >
+                    {(close: React.MouseEventHandler<HTMLButtonElement> | undefined) => (
+                        <div className='modal'>
+                            <Button className='close' onClick={close}>
+                                &times;
+                            </Button>
+                            <div className='header'>
+                                {' '}
+                                <h2> Disconnection </h2>
+                            </div>
+                            <div className='content'>
+                                <h3>
+                                    If you confirm your disconnection, you will leave the room and be redirected to the
+                                    home page.
+                                </h3>
+                            </div>
+                            <div className='actions'>
+                                <Button variant='contained' color='error' onClick={disconnect}>
+                                    Leave the room
+                                </Button>
+                                <Button variant='contained' onClick={close}>
+                                    Stay in the room
+                                </Button>
+                            </div>
+                        </div>
+                    )}
+                </Popup>
             </div>
-
-            <div className='DashboardButtonsRow'>
-                <div className='ButtonContainer'>
-                    <Button variant='outlined' className='Button' sx={{ borderRadius: 28 }}>
-                        <CameraswitchOutlinedIcon />
-                    </Button>
-                    Visualize Camera output
-                </div>
-                <div className='ButtonContainer'>
-                    <Button variant='outlined' className='Button' sx={{ borderRadius: 28 }}>
-                        <ForumOutlinedIcon />
-                    </Button>
-                    Chat
-                </div>
-                <div className='ButtonContainer'>
-                    <SliderComponent />
-                </div>
-                <div className='ButtonContainer'>
-                    <Button variant='contained' className='Button' color='error' sx={{ borderRadius: 28 }}>
-                        <ExitToAppOutlinedIcon />
-                    </Button>
-                    Disconnect
-                </div>
-            </div>
-
-            <div className='DashboardButtonsRow'></div>
         </div>
     );
 };
```

Comment:
This whole component is probably being modified at the same time by @ArthurNdy because he is working on the small screen version. For conflicts reasons, would it be possible to keep this component untouched, and put your modifications in a different component (for instance, `LargeDashboardButtonsComponent`) ? You can import this new component in the APP instead of this one, there will be less conflicts with @ArthurNdy branch